rir/src/compiler/debugging/stream_logger.cpp

```diff
@@ -198,5 +193,29 @@ void LogStream::section(const std::string& title) {
     highlightOff();
 }
 
+void StreamLogger::beginPhase(const std::string& msg) {
+    if (!options.includes(DebugFlag::PrintIntoFiles) &&
+        (options.intersects(PrintDebugPasses) ||
+         options.includes(DebugFlag::ShowWarnings))) {
+        std::cout << BLUE;
+        int l = 36 - (int)msg.length() / 2;
+        int r = l - msg.length() % 2;
+        std::cout << "\n╞";
+        for (int i = 0; i < l; ++i)
```

Comment:
`cout << setw(l) << "=";`

Reply:
ehm, that does not work.
It's `cout << std::setfill('=') << std::setw(l) << "" << std::setfill(' ') << "\n";`
honestly I am not convinced that this is nicer, plus it does not work in the case of the box drawing characters, since `setfill` only accepts an 8bit char.